extensions/amp-analytics/0.1/test/test-cookie-writer.js

```diff
@@ -109,8 +113,26 @@ describes.realWin('amp-analytics.cookie-writer', {
     it('Resolve when in inabox ad', () => {
       env.win.AMP_MODE.runtime = 'inabox';
       const config = dict({
-        'writeCookies': {
-          'testId': 'QUERY_PARAM(test)',
+        'cookies': {
+          'testId': {
+            'value': 'QUERY_PARAM(test)',
+          },
+        },
+      });
+      const cookieWriter = new CookieWriter(win, element, config);
+      expandAndWriteSpy = sandbox.spy(cookieWriter, 'expandAndWrite_');
```

Comment:
I am only stubbing the private method here. I did stub replacement service because I need to mock the value. 
I don't really see the benefit, but yes I can change to spy urlReplacementService.

Reply:
BTW I am stubbing `cookieWriter.urlReplacementService_` not the `urlReplacementService` anyway, because the urlReplacementService is installed on the top level doc, not the env.win. 
That to say I am not changing the stub logic here.